### DISCUSSION TITLE
What does a nuclear bomb actually sound like?

### DISCUSSION THREAD

**Discussion by RagexPrince683** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1141] [Show and tell]
A nuclear bomb, in reality sounds a lot like a large bang. Followed by a low rumble of air rushing.

https://youtu.be/szBRTPaoCM4

https://youtu.be/ruLt9FjL6gg

Could you please add a config option for realistic nuclear weapon sound effects, I personally think it would be cool.

**Comment by RadiumMarshmallows** [comment]
nuh uh

**Reply by BallOfEnergy1** [reply]
nuh uh

**Reply by thecodeconsumes** [reply]
yes it's a really cool idea

**Comment by ApexElite3303** [comment]
> A nuclear bomb, in reality sounds a lot like a large bang. Followed by a low rumble of air rushing.
> 
> https://youtu.be/szBRTPaoCM4
> 
> https://youtu.be/ruLt9FjL6gg
> 
> Could you please add a config option for realistic nuclear weapon sound effects, I personally think it would be cool.


Just manually replace the bombDet.ogg files

**Comment by RagexPrince683** [comment]
imo I think there should be a shockwave sound effect and the pitch should be much deeper

**Comment by Antineutrino-4444** [comment]
dude if u wanna do this type of simple stuff, just make a pr. It doesn’t require any coding whatsoever (I think so)
Posting in issues will only make people annoyed and does nothing other than that

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by RagexPrince683** [reply]
This is discussions. Not issues.

**Comment by Tiny-Mushroom** [comment]
Well, to make the blast actually sound realistic there would be some actual math involved. The distance from the blast would indicate how long it takes before the sound plays for you. And the yield of the bomb would determine the sound itself. Now I haven't had any experience in java but I think it wouldn't be that easy to write code to calculate what it would sound like, where and when the sound would play, and actually generate that sound cause the sample pool for actual nuclear blast audio is very small... I might start learning java just for this actually... or just go make some popcorn and watch a movie. Yeah second one is easier.

**Reply by RagexPrince683** [reply]
that sounds awesome, I'd love to see what you come up with.
